tests/models/test_models.py

```diff
@@ -0,0 +1,26 @@
+import json
+from pathlib import Path
+
+import pytest
+
+from fastapi_jwks.models.types import JWKS
+
+RESOURCE_PATH = Path(__file__).parent / "resources"
+
+
+@pytest.mark.parametrize(
+    "provider",
+    [
+        "google",  # https://www.googleapis.com/oauth2/v3/certs
+        "apple",  # https://developer.apple.com/documentation/signinwithapplerestapi/fetch_apple_s_public_key_for_verifying_token_signature
+        "microsoft",  # https://login.microsoftonline.com/common/discovery/v2.0/keys
+        "botframework",  # https://login.botframework.com/v1/.well-known/keys
+        "keycloak",  # Start a local keycloak and go to http://<keycloak>/realms/master/protocol/openid-connect/certs
+    ],
+)
+def test_jwks_model(provider: str):
+    """Test the JWKS model supports different providers"""
+    with open(RESOURCE_PATH / f"{provider}.json") as file:
+        jwks = json.loads(file.read())
+    validate = JWKS.model_validate(jwks)
```

Comment:
we could use builtin JSON support of Pydantic as well as the pathlib method to read file inline

```suggestion
    jwks = (RESOURCE_PATH / f"{provider}.json").read_text()
    validate = JWKS.model_validate_json(jwks)
```